packages/twenty-front/src/modules/ui/navigation/bread-crumb/components/Breadcrumb.tsx

```diff
@@ -14,7 +14,7 @@ const StyledWrapper = styled.nav`
   font-size: ${({ theme }) => theme.font.size.lg};
   font-weight: ${({ theme }) => theme.font.weight.semiBold};
   gap: ${({ theme }) => theme.spacing(2)};
-  line-height: ${({ theme }) => theme.text.lineHeight.md};
+  line-height: ${({ theme }) => theme.text.lineHeight.lg};
```

Comment:
Im aware this will change Breadcrumb lineheight everywhere but I think its a good thing as the Breadcrumb currently glitches when there is a save section on the same line:

https://github.com/twentyhq/twenty/assets/51697796/b6b9981b-020c-4cbd-baac-14ca2f6a3cef